src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -68,40 +70,38 @@ public void ChatTestsStaff_002_verifySwitchingBetweenMainAndPrivateSections() {
   @DontRun(env = {"sandbox"})
   @Test(groups = {"ChatTestsStaff_003", "ChatStaff"})
   public void ChatTestsStaff_003_sendPrivateMessage() {
-    switchToWindow(driverOne);
     ChatPageObject chatUserOne =
-        openChatForUser(driverOne, credentials.userNameStaff, credentials.passwordStaff);
+        openChatForUser(credentials.userNameStaff, credentials.passwordStaff);
+
+    switchToWindow(1);
+    new SpecialVersionPage().open();
 
-    switchToWindow(driverTwo);
     ChatPageObject chatUserTwo =
-        openChatForUser(driverTwo, credentials.userNameStaff2, credentials.passwordStaff2);
+        openChatForUser(credentials.userNameStaff2, credentials.passwordStaff2);
 
     String userTwoPublicMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
     chatUserTwo.writeOnChat(userTwoPublicMessage);
 
-    switchToWindow(driverOne);
+    switchToWindow(0);
     chatUserOne.verifyMessageOnChat(userTwoPublicMessage);
 
-    switchToWindow(driverTwo);
+    switchToWindow(1);
     String userTwoPrivateMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
     chatUserTwo.selectPrivateMessageToUser(credentials.userNameStaff);
     chatUserTwo.writeOnChat(userTwoPrivateMessage);
 
-    switchToWindow(driverOne);
+    switchToWindow(0);
     chatUserOne.verifyPrivateMessageHeader();
     chatUserOne.verifyPrivateMessageNotification();
     chatUserOne.clickOnUserInPrivateMessageSection(credentials.userNameStaff2);
     chatUserOne.verifyMessageOnChat(userTwoPrivateMessage);
   }
 
-  @DontRun(env = {"prod", "sandbox"})
+  @DontRun(env = {"prod"})
   @Test(groups = {"ChatTestsStaff_004", "ChatStaff"})
   public void ChatTestsStaff_004_basicUserChatFails() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)